datadog_checks_base/datadog_checks/checks/openmetrics/mixins.py

```diff
@@ -96,10 +83,6 @@ def create_scraper_configuration(self, instance=None):
 
         config['metrics_mapper'] = metrics_mapper
 
-        # `rate_metrics` contains the metrics that should be sent as rates
-        config['rate_metrics'] = self._extract_rate_metrics(default_instance.get('type_overrides', {}))
```

Comment:
If we're getting rid of the `type_overrides` config option, we will probably be changing expected behavior for some customers' checks.